### DISCUSSION TITLE
Documentation for customizing deployments in 4.0.2

### DISCUSSION THREAD

**Discussion by Dirk-Manderin** [PSAppDeployToolkit/PSAppDeployToolkit#1128] [Q&A] [answered]
The [Customizing Deployments](https://psappdeploytoolkit.com/docs/usage/customizing-deployments) section of the docs still shows using the AppDeployToolkitConfig.xml file.  I believe that is no longer used in v4, but I don't see anything about the new way in v4.  Is the documentation missing, or am I just not seeing it?

**Comment by dspencer42** [comment]
For our endpoint team, we use our own modified template of PSADT hosted on Git so the whole team can use the same universal template building apps for our organization.

I will have everyone install the PSADT Module, but instead of using New-ADTTemplate just clone the one from our Git back down to them.

The logo and banner files are in the Assets directory and can be updated with your own.
Other custom config and options such as logging targets are in the .\Config\config.psd1 file... which is duplicated at .\PSAppDeployToolkit\config\config.psd1

**Reply by FRacine** [reply]
It is unclear for me too.

**Comment by DanGough** [comment]
There are indeed a few things still outstanding with the docs. Apologies for this, but site updates are in the pipeline!

Editing the .psd1 files in the Config/Strings folders from the deployment template is the recommended approach.

**Comment by dancing-groot** [comment]
I'm scratching my head as well - Ideally I'd want the module installed on the devices, with the minimum amount of files packaged with the app. But, at the moment, I'm drawing a blank.

**Comment by DanGough** [comment]
- Either grab one of the zipped templates, or generate one with `New-ADTTemplate`.
- This has config/assets/strings folders with copies of the config files ready for you to modify.
- If you want to keep the package size down and pre-deploy the PSAppDeployToolkit module to your devices, all you need to do is delete the PSAppDeployToolkit folder from this template and it will fall back to the locally installed one.

**Reply by dancing-groot** [reply]
Hi Dan,

I have tried that. The EXE complains that it can't find the module folder. The PS1 does seem to launch correctly.
FWIW this is what the folder looks like.
![Screenshot 2024-12-09 171936](https://github.com/user-attachments/assets/d25b2f98-fe0d-49fe-89e0-521ef1023db9)

**Reply by DanGough** [reply]
Can you retry with the latest version 4.0.3?

**Reply by dancing-groot** [reply]
That's the version I'm using, only started looking at v4 today.

**Comment by mjr4077au** [comment, marked as answer]
The "Customising Deployments" documentation has since been updated (https://psappdeploytoolkit.com/docs/usage/customizing-deployments). If there's anything still unclear, please raise it as an issue so we can amend the documentation further.
